jobs/worker/spec

```diff
@@ -172,15 +167,14 @@ properties:
     description: |
       Disable remapping of user/group IDs in unprivileged volumes.
 
-      For use in combination with 'garden.use_houdini'.
+      For use in combination with 'runtime.type=houdini'.
     default: false
 
-  garden.use_houdini:
-    env: CONCOURSE_GARDEN_USE_HOUDINI
```

Comment:
Can we leave this for backwards compatibility, making it an alias for `runtime.type=houdini`? I'm not sure if there's a tidy way to get bosh to output deprecation warnings, but it's always nice to make a smoother upgrade experience. I imagine this only gets used in pretty weird legacy situations (i.e. you're able to deploy a linux VM with bosh but you can't run containers? i don't know if bosh even allows for locked-down setups like this... maybe if your VM was a real snowflake with lots of important hardware attached that your tasks needed direct access to? I'm rambling) so if it's at all tricky to do, feel free to push back on this ask.

sounds like cloudfoundry/bosh#1904 is the kind of (nonexistent) thing I was hoping for.

Reply:
@jamieklassen hm we could perhaps alias it by keeping the field in the spec then converting it like this in the template:

```
<% if_p("garden.use_houdin") do |v| -%>
export CONCOURSE_RUNTIME_TYPE="houdini"
<% end -%>
```

However, this would mean updating https://github.com/concourse/ci/blob/master/tasks/scripts/check-distribution-env/bosh/ignored-in-distribution to include `CONCOURSE_GARDEN_USE_HOUDINI`. I know it's only one line but those `ignored-in` files are a bit painful to manage atm. One of the potential benefits of the separate flags is to make that a bit easier to reason about. So I would prefer to keep this out, and make a clean break.